homeworks/sasha.demidenko_SashaDemi/GamesProject/About.css

```diff
@@ -0,0 +1,75 @@
+.about_body {
```

Comment:
Please change the names of the files:
- About.html -> index.html (and correct links in header)
- Games.html -> games.html
all styles put to folder `css` or `styles`